python/nav/statemon/db.py

```diff
@@ -75,12 +78,12 @@ def connect(self):
             self.db = psycopg2.connect(conn_str)
             atexit.register(self.close)
 
-            debug("Successfully (re)connected to NAVdb")
+            LOGGER.info("Successfully (re)connected to NAVdb")
             # Set transaction isolation level to READ COMMITTED
             self.db.set_isolation_level(1)
         except Exception as err:
-            debug("Couldn't connect to db.", 2)
-            debug(str(err), 2)
+            LOGGER.critical("Couldn't connect to db.")
+            LOGGER.critical(str(err))
```

Comment:
Hmm. The debug-system had several levels I translated to just `critical`. Every case of `critical` followed by `str(err)`-critical should maybe use `exception` instead?